sh.exe.stackdump

```diff
@@ -0,0 +1,28 @@
+Exception: STATUS_ACCESS_VIOLATION at rip=002100C9869
+rax=0000000000000000 rbx=0000000210359A08 rcx=0000000000000040
```

Comment:
What is this file?

Reply:
A file I've been trying to keep out of my commits. Not sure what it is, but I'll remove it.